features/raadpleeg-verblijfplaats-met-peildatum/gba/test/geheimhouding-gba.feature

```diff
@@ -0,0 +1,87 @@
+      | naam                     | waarde |
+      | indicatie geheim (70.10) | 0      |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2023-01-01            |
+      Dan heeft de response verblijfplaatsen met de volgende gegevens
+      | gemeenteVanInschrijving.code | gemeenteVanInschrijving.omschrijving | datumAanvangAdreshouding | straat           |
+      | 0800                         | Hoogeloon, Hapert en Casteren        | 20220801                 | Korte straatnaam |
+
+  Rule: indicatie geheim met waarde hoger dan 0 wordt ongevraagd meegeleverd
+
+    Scenario: historie wordt gevraagd op burgerservicenummer met geheimhouding <geheimhouding persoonsgegevens>
+      Gegeven de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                     | waarde                           |
+      | indicatie geheim (70.10) | <geheimhouding persoonsgegevens> |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2023-01-01            |
+      Dan heeft de response verblijfplaatsen met de volgende gegevens
+      | gemeenteVanInschrijving.code | gemeenteVanInschrijving.omschrijving | datumAanvangAdreshouding | straat           |
+      | 0800                         | Hoogeloon, Hapert en Casteren        | 20220801                 | Korte straatnaam |
+      En heeft de response de volgende gegevens
+      | naam                          | waarde                           |
+      | geheimhoudingPersoonsgegevens | <geheimhouding persoonsgegevens> |
+
+      Voorbeelden:
+      | geheimhouding persoonsgegevens |
+      | 1                              |
+      | 2                              |
+      | 3                              |
+      | 4                              |
+      | 5                              |
+      | 6                              |
+      | 7                              |
+
+    Scenario: historie wordt gevraagd van persoon met geheimhouding op peildatum waarop geen verblijfplaats wordt gevonden
+      Gegeven de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                     | waarde |
+      | indicatie geheim (70.10) | 7      |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2022-01-01            |
+      Dan heeft de response geen 'verblijfplaatsen' gegevens
```

Comment:
Dit is nu niet consistent met 'response 0 verblijfplaatsen'. De tekst 'response geen 'verblijfplaatsen' gegevens' wordt nu ook niet ondersteund in de stepdefs.js

Reply:
ik heb de scenario's nu overal aangepast, zodat alleen nog "Dan heeft de response 0 verblijfplaatsen" wordt gebruikt